test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -275,6 +289,62 @@ protected void StartOcelot(Action<WebHostBuilderContext, IConfigurationBuilder>
         _ocelotClient = _ocelotServer.CreateClient();
     }
 
+    /// <summary>
+    /// This is annoying cos it should be in the constructor but we need to set up the file before calling startup so its a step.
+    /// </summary>
+    /// <typeparam name="T">The <see cref="ILoadBalancer"/> type.</typeparam>
+    /// <param name="loadBalancerFactoryFunc">The delegate object to load balancer factory.</param>
+    public void GivenOcelotIsRunningWithCustomLoadBalancer<T>(Func<IServiceProvider, DownstreamRoute, IServiceDiscoveryProvider, T> loadBalancerFactoryFunc)
+        where T : ILoadBalancer
+    {
+        _webHostBuilder = new WebHostBuilder()
+            .ConfigureAppConfiguration((hostingContext, config) =>
+            {
+                config.SetBasePath(hostingContext.HostingEnvironment.ContentRootPath);
+                var env = hostingContext.HostingEnvironment;
+                config.AddJsonFile("appsettings.json", true, false)
+                    .AddJsonFile($"appsettings.{env.EnvironmentName}.json", true, false);
+                config.AddJsonFile(_ocelotConfigFileName, false, false);
+                config.AddEnvironmentVariables();
+            })
+            .ConfigureServices(s =>
+            {
+                s.AddOcelot()
+                    .AddCustomLoadBalancer(loadBalancerFactoryFunc);
+            })
+            .Configure(async app => { await app.UseOcelot(); });
+
+        _ocelotServer = new TestServer(_webHostBuilder);
+        _ocelotClient = _ocelotServer.CreateClient();
+    }
+
+    public void GivenOcelotIsRunningWithConsul(params string[] urlsToListenOn)
```

Comment:
It appears to have no references, correct? We have ceased using this method in favor of the `GivenOcelotIsRunningWithServices` method.
### Please remove it❕

Reply:
Done!